frontend/integration-tests/tests/login.scenario.ts

```diff
@@ -0,0 +1,74 @@
+import { $, browser, ExpectedConditions as until } from 'protractor';
+import { appHost } from '../protractor.conf';
+import * as loginView from '../views/login.view';
+import * as sidenavView from '../views/sidenav.view';
+import * as clusterSettingsView from '../views/cluster-settings.view';
+
+const JASMINE_DEFAULT_TIMEOUT_INTERVAL = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+const JASMINE_EXTENDED_TIMEOUT_INTERVAL = 1000 * 60 * 3;
+const {
+  CLUSTER_READER_IDP ='test',
+  CLUSTER_READER_USERNAME = 'test',
+  CLUSTER_READER_PASSWORD = 'test',
+  KUBEADMIN_IDP = 'kube:admin',
+  KUBEADMIN_USERNAME = 'kubeadmin',
```

Comment:
`KUBEADMIN_IDP` and `KUBEADMIN_USERNAME` shouldn't need to be env variables. Those never change.